Sources/Hummingbird/Middleware/FileMiddleware.swift

```diff
@@ -52,6 +52,7 @@ where Provider.FileAttributes: FileMiddlewareFileAttributes {
     let searchForIndexHtml: Bool
     let urlBasePath: String?
     let fileProvider: Provider
+    public let mediaTypeFileExtensionMap: [String: MediaType]
```

Comment:
Is there a reason this needs to be public.

Reply:
re @Joannis 
>it's certainly also no harm and potentially helpful to know all registered file extensions.

I thought it was a good point. I could see a need to introspect what the customizations are.

Reply:
Until someone needs it I'd prefer we leave it private. Setting it to public means setting it in stone until the next major version.